apps/crn-server/src/data-providers/contentful/users.data-provider.ts

```diff
@@ -43,6 +45,15 @@ export type TeamMembership = NonNullable<
   NonNullable<UserItem['teamsCollection']>['items'][number]
 >;
 
+type OrcidWorkContentful = {
+  id: string;
+  doi?: string;
+  title?: string;
+  type?: string;
+  publicationDate?: Record<'day' | 'month' | 'year', string | undefined>;
+  lastModifiedDate?: string;
+};
```

Comment:
Isn't this pretty much the same as 

```
export type OrcidWorkCMS = {
  id: string;
  doi?: string;
  title?: string;
  type?: string;
  publicationDate?: {
    day?: string;
    month?: string;
    year?: string;
  };
  lastModifiedDate?: string;
};
```
from entities file?

Reply:
it is, though the idea is that we declare it's own type in the data provider and then in the entities we have declared a uniform type that happens to work for both. In fact because these structures are both JSON in squidex and contentful they are pretty much identical, but i think the whole point is to still keep them separate.